### PR TITLE
HSD8-536 Accordion pattern and paragraph type

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Accordion pattern and paragraph type
- uses drupal core collapse javascript for accessibility and IE help.

# Needed By (Date)
- End of the sprint. 2/13

# Urgency
- Low

# Steps to Test

1. Checkout this branch
1. sync a site `blt drupal:sync --site=[sitename]`
1. create a basic page and add an accordion component
1. ensure it functions as expected.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: docroot/themes/humsci/su_humsci_theme/patterns/molecules/accordion/accordion.html.twig [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{% if variant == 'open' %}
+  {% set attributes = attributes.setAttribute('open', 'open') %}
+{% endif %}
+
+<details{{ attributes }}>
+  {%- if title -%}
```
Comment (sherakama) [inline docroot/themes/humsci/su_humsci_theme/patterns/molecules/accordion/accordion.html.twig:6]: TIL: Whitespace control at the tag level. Neato. 
I've been tempted to add {% spaceless %} around the whole html.html.twig document but this is nice as well.

Comment (pookmish) [reply docroot/themes/humsci/su_humsci_theme/patterns/molecules/accordion/accordion.html.twig:6]: Yeah i grabbed this template from `stable` theme and had to look that up.

Comment (sherakama) [reply docroot/themes/humsci/su_humsci_theme/patterns/molecules/accordion/accordion.html.twig:6]: Oh, this is from stable? Can you not just extend that template here then?

Comment (pookmish) [reply docroot/themes/humsci/su_humsci_theme/patterns/molecules/accordion/accordion.html.twig:6]: stable doesnt have blocks and i needed to wrap `description` and `children` in a wrapper.

Comment (sherakama) [reply docroot/themes/humsci/su_humsci_theme/patterns/molecules/accordion/accordion.html.twig:6]: bummer man,

File: docroot/themes/humsci/su_humsci_theme/patterns/molecules/accordion/js/accordion.polyfill.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+(function ($, Drupal) {
```
Comment (sherakama) [inline docroot/themes/humsci/su_humsci_theme/patterns/molecules/accordion/js/accordion.polyfill.js]: Remove this empty file.

File: docroot/themes/humsci/su_humsci_theme/su_humsci_theme.theme [hunk 3]
```diff
@@ -9,6 +9,19 @@ use Drupal\views\Plugin\views\field\EntityField;
 use Drupal\Component\Utility\Html;
 use Drupal\node\Entity\Node;
 use Drupal\Component\Utility\SortArray;
+use Drupal\Core\Template\Attribute;
+
+function su_humsci_theme_preprocess_pattern_accordion(&$variables) {
```
Comment (sherakama) [inline docroot/themes/humsci/su_humsci_theme/su_humsci_theme.theme:17]: Missing function docblock.

Comment (pookmish) [reply docroot/themes/humsci/su_humsci_theme/su_humsci_theme.theme:17]: huh. why didnt CC report that

Comment (sherakama) [reply docroot/themes/humsci/su_humsci_theme/su_humsci_theme.theme:17]: probably because it is a `.theme` file.

**Review by sherakama — APPROVED** [review]
This works well. 

I had issues with patches in the composer.json file but that is unrelated to this and is ok to merge with the two changes requested.

**Review by sherakama — APPROVED** [review]

**Pull request merged**